backend/src/index.ts

```diff
@@ -0,0 +1,50 @@
+import dotenv from "dotenv";
+import express, { Request, Response } from "express";
+import cors from "cors";
+import helmet from "helmet";
+import http from "http";
+import { Server as IOServer, Socket } from "socket.io";
+
+dotenv.config();
+
+// Setup Environment Variables
+if (!process.env.PORT) {
+  console.error('ERROR: Environment variable "PORT" not set');
```

Comment:
Isn't the convention to have something like this: `const PORT = process.env.PORT || 3001;`?

As it is, having the app erroring and not starting without the env var set when we could have a sensible default seems like it could be frustrating.

Reply:
I've just implemented [convict](https://github.com/mozilla/node-convict/tree/master/packages/convict) in d563ebf to manage our config for us which handles defaults now!